pertpy/tools/_metadata/_cell_line.py

```diff
@@ -78,217 +90,174 @@ def __init__(self):
-
-    def getinfo_annotate_ccle_expression(self) -> None:
-        """A brief summary of CCLE expression data."""
-        print(f" Expression of {len(self.ccle_expr.columns.unique())} genes is saved in this file.")
-        print(f"{len(self.ccle_expr.index.unique())} unique cell lines are available in this file.")
-
     def annotate_cell_lines(
         self,
         adata: AnnData,
-        cell_line_identifier: str = "DepMap_ID",
-        identifier_type: str = "DepMap_ID",
-        cell_line_information: list[str] = None,
-        source: str = "DepMap",
+        query_id: str = "DepMap_ID",
+        reference_id: str = "DepMap_ID",
+        cell_line_information: list[str] | None = None,
+        cell_line_source: Literal["DepMap", "Cancerrxgene"] = "DepMap",
         copy: bool = False,
     ) -> AnnData:
         """Fetch cell line annotation.
 
         For each cell, we fetch cell line annotation from Dependency Map (DepMap).
 
         Args:
-            adata: The AnnData object to annotate.
-            cell_line_identifier: The column of `.obs` with cell line information. Defaults to "DepMap_ID".
-            identifier_type: The type of cell line information, e.g. DepMap_ID, cell_line_name or
-                stripped_cell_line_name. To fetch cell line metadata from Cancerrxgene, it is recommended to choose
+            adata: The data object to annotate.
+            query_id: The column of `.obs` with cell line information. Defaults to "DepMap_ID".
+            reference_id: The type of cell line identifier in the meta data, e.g. DepMap_ID, cell_line_name or
+                stripped_cell_line_name. If fetch cell line metadata from Cancerrxgene, it is recommended to choose
                 "stripped_cell_line_name". Defaults to "DepMap_ID".
             cell_line_information: The metadata to fetch. All metadata will be fetched by default. Defaults to None (=all).
-            source: The source of cell line metadata, DepMap or Cancerrxgene. Defaults to "DepMap".
-            copy: Determines whether a copy of the AnnData object is returned. Defaults to False.
+            cell_line_source: The source of cell line metadata, DepMap or Cancerrxgene. Defaults to "DepMap".
+            copy: Determines whether a copy of the `adata` is returned. Defaults to False.
 
         Returns:
             Returns an AnnData object with cell line annotation.
         """
         if copy:
             adata = adata.copy()
 
-        if source == "DepMap":
+        if cell_line_source == "DepMap":
             cell_line_meta = self.cell_line_meta
-        elif source == "Cancerrxgene":
-            cell_line_meta = self.cl_cancer_project_meta
         else:
-            raise ValueError(
-                "The specified source of cell line metadata is not available."
-                "Please choose either DepMap or Cancerrxgene. Default is DepMap."
-            )
+            if query_id == "DepMap_ID":
+                print(
```

Comment:
Ahhh, so you have such a warning! I was complaining about this in the notebook. Why did I not see this? But can't you just continue running this function with the correct reference identifier?